app/assets/stylesheets/header-footer-only.scss

```diff
@@ -3,6 +3,8 @@ $govuk-use-legacy-palette: false;
 
 @import "govuk_publishing_components/govuk_frontend_support";
 @import "govuk_publishing_components/components/_cookie-banner";
+// collections uses the action-link component and relies upon this line, see
+// https://github.com/alphagov/collections/pull/1754
```

Comment:
I'd suggest adding a space between the comment and the line it does not apply to (I think the line above) as that would make it clearer that the comment applies to the line below and not the line above.